runtime/trappist/src/xcm_config.rs

```diff
@@ -230,7 +230,7 @@ parameter_types! {
 		0u128
 	);
 	/// Roc = 7 RUSD
-	pub RocPerSecond: (xcm::v3::AssetId, u128,u128) = (MultiLocation::parent().into(), default_fee_per_second() * 70, 0u128);
+	pub RocPerSecond: (xcm::v3::AssetId, u128,u128) = (MultiLocation::new(1,Here).into(), default_fee_per_second() * 70, 0u128);
```

Comment:
Shouldn't `fmt` catch things like this?
```suggestion
	pub RocPerSecond: (xcm::v3::AssetId, u128, u128) = (MultiLocation::new(1, Here).into(), default_fee_per_second() * 70, 0u128);
```

Reply:
Recently a few formatting rules were changed since they were only available in nightly versions of Rust. That might be the reason.